packages/mdc-select/README.md

```diff
@@ -269,6 +269,23 @@ If you'd like to maintain the width or height outside of the attribute, you'll n
 
 It is advised that dividers also set `role="presentation"` to disable selection and not cloud accessibility.
 
+### Sass Mixins
+
+To customize the colors of any part of the select, use the following mixins. We recommend you use 
+these mixins within CSS selectors like `.foo-select` to apply styling. 
+
+Mixin | Description
+--- | ---
+`mdc-select-ink-color($color)` | Customizes the text entered into the select.* 
+`mdc-select-fill-color($color)` | Customizes the background color of the select. 
+`mdc-select-label-color($color)` | Customizes the label color of the select in the unfocused state. 
+`mdc-select-focused-label-color($color, $opacity: 0.87)` | Customizes the label color of the select when focused. Changing opacity for the label when floating is optional.
+`mdc-select-bottom-line-color($color)` | Customizes the color of the default bottom line of the select.
+`mdc-select-focused-bottom-line-color($color)` | Customizes the color of the bottom line of the select when focused.
+`mdc-select-selected-text-color($color)` | Customizes the color of the option that has been selected. 
```

Comment:
Can you attach a screenshot of what you are coloring here? I cant figure out what this is.

Reply:
<img width="218" alt="screen shot 2018-01-11 at 3 40 38 pm" src="https://user-images.githubusercontent.com/34519388/34852909-e12de4b4-f6e5-11e7-934c-f62c4afc4682.png">

The "Fruit Roll Ups" text is the `mdc-select-selected-text-color`

Reply:
Try and combine this with mdc-select-ink-color. And mdc-select-ink-color is the better name.